wiki/Storyboard/Scripting/en.md

```diff
@@ -1,34 +1,41 @@
 # Storyboard scripting
 
-![An example of scripting in .osb](img/SBS_Base.jpg "An example of scripting in .osb")
+![Screenshot of a storyboard script open in "Code - OSS"](img/osb-example.png "Example of a storyboard script (.osb)")
 
-In addition to the [built-in storyboard editor](/wiki/Client/Beatmap_editor/Design), a plain text editor, such as Notepad, [Notepad++](https://www.notepad-plus-plus.org/), or [Visual Studio Code](https://code.visualstudio.com/), can also be used to create [storyboards](/wiki/Storyboard) for beatmaps. This workflow is called **storyboarding by scripting** (*SBS*) and it allows one to be more precise (to the pixel and the millisecond) as to when sprites appear, how they behave, and so on.
+**Storyboard scripting** is the process of editing osu! [storyboards](/wiki/Storyboard) via their [`.osb`](/wiki/Client/File_formats/Osb_(file_format)) and [`.osu`](/wiki/Client/File_formats/Osu_(file_format)) files. These files define images and effects that the [game client](/wiki/Client) renders into background animations during gameplay.
 
-To study a specific part of scripting, head to one of the articles below:
+Scripting is a common alternative to [the game client's storyboard editor](/wiki/Client/Beatmap_editor/Design) when precision or complexity are required. With a text editor such as [Notepad](https://en.wikipedia.org/wiki/Windows_Notepad "\"Windows Notepad\" on Wikipedia"), [Notepad++](https://www.notepad-plus-plus.org/), or [Visual Studio Code](https://code.visualstudio.com/), storyboarders can edit the exact values of [objects](Objects) and [commands](Commands) instead of using on-screen controls. Many storyboarders also make use of [programming](https://en.wikipedia.org/wiki/Computer_programming "\"Computer programming\" on Wikipedia") to build storyboard files that would be difficult to create manually.
 
-- Basic concepts:
-  - [General rules of storyboarding](General_Rules)
-  - [Objects](Objects)
-  - [Basic commands](Commands)
-  - [Compound commands](Compound_Commands)
-  - [Variables](Variables)
-- Extra:
-  - [Playing audio](Audio)
-  - [Storyboard scripting cheat sheet](Cheat_Sheet)
-  - [osu! file toggles](osu!_File_Toggles)
-  - [Short versions of commands](Shorthand)
+*Warning: Avoid editing storyboard scripts of a [beatmap](/wiki/Beatmap) currently open in the [beatmap editor](/wiki/Client/Beatmap_editor). Your work on scripts may be overwritten when saving the beatmap.*
```

Comment:
it is slightly redundant since the warning is about the scripts anyway

```suggestion
*Warning: Avoid editing storyboard scripts of a [beatmap](/wiki/Beatmap) currently open in the [beatmap editor](/wiki/Client/Beatmap_editor). Your work may be overwritten when saving the beatmap.*
```